woof-code/rootfs-skeleton/usr/local/petget/installpkg.sh

```diff
@@ -813,7 +813,9 @@ if [ "$xpkgname" != "" ] && [ "$installed_pkg" != "" ]; then
 	 fi
 	done < /var/packages/${installed_files}.files
 	
-	rm -f /var/packages/${installed_files}.files
+	if [ "${DLPKG_NAME}.files" != "${installed_files}.files" ]; then
```

Comment:
But how and why a new packages file and an existing package file will have the same filename???

Reply:
Because some users renamed the packages for some reasons instead of leaving the package filename alone